index.html

```diff
@@ -211,7 +212,29 @@ <h3 class="about-title">About me</h3>
             </section>
                 
         </article>
-
+        <article id="contact-form"> 
+          <section class="contact-flex">  
+            <h2 class="contact-title">I'm always <br/>interested in hearing about new projects,<br/> so if you'd like to <br/>chat please get in<br/> touch.</h2>
+            <form action="https://formspree.io/f/xvonwpvygit" method="POST">
+                <ul class="contact-inputs">
+                    <li><input type="text" placeholder="Full name" name="full name" required></li>
+                    <li><input type="email" placeholder="Email Address" name="email" required></li>
+                    <li><textarea placeholder="write me something ...."  name="message" required></textarea></li>
+                    <li class="form-button"><button type ="submit" class="view-button">Get in touch</button></li>  
+                </ul>
```

Comment:
- [ ] Kindly, your form is not successfully integrated with [formspree service](https://formspree.io/html). Please check well your ``` form id``` and make sure that it is being submitted successfully 🙏 

### Currently what I get after submitting 👇 
![image](https://user-images.githubusercontent.com/102757126/228255962-e1b51c40-181c-49f0-b241-3b3652b1607c.png)

- [ ] Please as recommended in the project requirements, our name input should accept up to **30 characters**, and Message input should have ```maxlength="500"``` to avoid invalid input and unlimited characters. Kindly add these validations as suggested in the project requirements 😁😁
- [ ] As we have used CSS Flexbox to layout our elements, it would be better to remove the ```<ul> and <li>``` tags to keep your codes clean and professional 😎😎